spec/buftok_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'filewatch/buftok'
+
+describe FileWatch::BufferedTokenizer do
+
+  context "when using the default delimiter" do
+    subject { FileWatch::BufferedTokenizer.new }
+
+    it "splits the lines correctly" do
+      message = "hello\nworld\n"
+      expect(subject.extract(message)).to eq ["hello", "world"]
+    end
+  end
+
+  context "when passing a custom delimiter" do
+    subject { FileWatch::BufferedTokenizer.new("\r\n") }
+
+    it "splits the lines correctly" do
+      message = "hello\r\nworld\r\n"
+      expect(subject.extract(message)).to eq ["hello", "world"]
```

Comment:
it's a different semantic, from what I understand

  expect([1, 2, 3]).to    contain_exactly(2, 3, 1) # pass

but I want to make sure the order is the one I specified, so this won't work.